python/cudf/cudf/io/parquet.py

```diff
@@ -546,7 +546,15 @@ def _parse_metadata(meta) -> tuple[bool, Any, None | np.dtype]:
 @_performance_tracking
 def read_parquet_metadata(
     filepath_or_buffer,
-) -> tuple[int, int, list[Hashable], int, list[dict[str, int]]]:
+) -> tuple[
+    int,
+    int,
+    list[int],
+    list[Hashable],
+    int,
+    list[dict[str, int]],
+    dict[str, list[dict[str, int]]],
+]:
```

Comment:
This is a breaking change.

Reply:
Yes, it is. Do we not want to expose these from the python api?

Reply:
Oh I didn't realize you already marked it "breaking" (I was just making you aware).
> Do we not want to expose these from the python api?

But now that you mention it, we shouldn't because we don't need it. If we did include it, I think we'd need to follow our deprecation process.

Reply:
All done!